docker-resources/docker-compose-base-test.yml

```diff
@@ -46,3 +46,4 @@ services:
 #   url: jdbc:postgresql://localhost:3336/plugin-valtimo-test
 #   url: jdbc:postgresql://localhost:3337/verzoek-test
 #   url: jdbc:postgresql://localhost:3338/portaaltaak-test
+#   url: jdbc:postgresql://localhost:3339/search-test
```

Comment:
It looks like port 3339 has been taken

Reply:
fixed in this pr: https://github.com/valtimo-platform/valtimo-backend-libraries/pull/504